src/spells/combatspells.c

```diff
@@ -882,7 +882,7 @@ int flee_spell(struct castorder * co, int strength)
         return 0;
     }
 
-    fgs = select_fighters(b, fi->side, FS_ENEMY, select_afraid, NULL);
+    fgs = select_fighters(b, fi->side, FS_ENEMY, wounded?select_afraid:select_alive, NULL);
```

Comment:
Wenn das Problem ist, dass man in Präkampfzaubern keine Verwundeten "bevorzugen" kann, könnte man diese Auswahl hier nicht alternativ an `(co->sp->sptyp & PRECOMBATSPELL)` knüpfen?

Reply:
Das es den Typ gibt, war mir nicht klar. Ja, klingt richtig. Man will in Präkampfzaubern nicht auf verwundete sortieren, weil es da noch keine gibt (verwundet ist hier nicht im Sinne von HP).

Was ich übrigens noch sagen wollte: Die "aura_of_fear", die das hier auch benutzt hängt anscheinend nur an der Rasse shadow_dragon und es scheint, dass die Rasse ausgestorben ist. Jedenfalls wird sie nirgends erzeugt. Da ist also evtl. toter Code, falls die nicht noch irgendwo in den Daten leben.

Reply:
Interessant. Gucke ich mir mal an.

Reply:
Den Schattendrachen gibt es wohl wirklich nicht (mehr), aber der Zauber wurde in E3 verwendet: 
https://wiki.eressea.de/index.php/Zauberbeschreibungen_E3#Furchteinfl.C3.B6.C3.9Fende_Aura